WcaOnRails/config/locales/en.yml

```diff
@@ -97,15 +104,16 @@ en:
       name: "Combined qualification"
       cellName: "Combined qualification"
   round:
-    name: "%{event} Round %{number}"
+    #context: e.g. event_name can be "3x3x3 Cube", and round_name can be "Combined First round"
+    name: "%{event_name}, %{round_name}"
```

Comment:
I think it may differ across translations.
I actually used a coma because it would make sense in French to put one there :p
I'm fine with removing it from English, though.